nemo/collections/tts/models/degli.py

```diff
@@ -0,0 +1,331 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Does this file contain code from https://github.com/Sytronik/deep-griffinlim-iteration?

Reply:
yes.

Reply:
Can you add their MIT license to this file? Like https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/tts/modules/squeezewave.py